lib/lettering/font.py

```diff
@@ -103,28 +108,58 @@ def _check_variants(self):
 
     name = localized_font_metadata('name', '')
     description = localized_font_metadata('description', '')
-    default_variant = font_metadata('default_variant', FontVariant.LEFT_TO_RIGHT)
-    default_glyph = font_metadata('defalt_glyph', u"�")
+    default_glyph = font_metadata('default_glyph', "�")
```

Comment:
I think this really does need to be a unicode string?

Reply:
Oh yes, right. I am using the new version to test, we don't need to do that there.